### ISSUE TITLE
debian glibc 2.31-1 is missing wcwidth() for many unicode 13 characters

### ISSUE THREAD

**Issue by dankamongmen** [dankamongmen/notcurses#1211]
see #1210. at least the following characters have an invalid `wcwidth()` on debian unstable's glibc:

```
[schwarzgerat](0) $ ./wcwidth ▔🮂🮃▀🮄🮅🮆█
0x02594: 1 ▔	0x1fb82: -1 🮂	0x1fb83: -1 🮃	0x02580: 1 ▀	
0x1fb84: -1 🮄	0x1fb85: -1 🮅	0x1fb86: -1 🮆	0x02588: 1 █	

 Total width: 3  total bytes: 29

[schwarzgerat](0) $ ./wcwidth 🮇🮇🮈▐🮉🮊🮋█
0x1fb87: -1 🮇	0x1fb87: -1 🮇	0x1fb88: -1 🮈	0x02590: 1 ▐	
0x1fb89: -1 🮉	0x1fb8a: -1 🮊	0x1fb8b: -1 🮋	0x02588: 1 █	

 Total width: 2  total bytes: 30

[schwarzgerat](0) $ ./wcwidth 🬞🬦▐🬏🬭🬵🬷🬓🬱🬹🬻▌🬲🬺█
0x1fb1e: -1 🬞	0x1fb26: -1 🬦	0x02590: 1 ▐	0x1fb0f: -1 🬏	
0x1fb2d: -1 🬭	0x1fb35: -1 🬵	0x1fb37: -1 🬷	0x1fb13: -1 🬓	
0x1fb31: -1 🬱	0x1fb39: -1 🬹	0x1fb3b: -1 🬻	0x0258c: 1 ▌	
0x1fb32: -1 🬲	0x1fb3a: -1 🬺	0x02588: 1 █	
 Total width: 3  total bytes: 57

[schwarzgerat](0) $ 
```

we work around the sextant problem by using direct writes from the blitters, but the missing eighths is about to cause us a real problem in progbars. figure out whether this is fixed upstream, and if not, submit a patch. if it is, figure out why debian's sucks.

**Comment by dankamongmen** [comment]
https://www.mail-archive.com/gcc-patches@gcc.gnu.org/msg246988.html

**Comment by dankamongmen** [comment]
ok, it looks like this was fixed in glibc 2.32: https://sourceware.org/pipermail/libc-announce/2020/000029.html

```
* Unicode 13.0.0 Support: Character encoding, character type info, and
  transliteration tables are all updated to Unicode 13.0.0, using
  generator scripts contributed by Mike FABIAN (Red Hat).
```

well, crappo.

**Comment by dankamongmen** [comment]
I don't see what we can do beyond waiting for Debian Unstable to move to glibc 2.32.

As for the vertical progbar problem mentioned in #1210, we've worked around this by directly blitting.
